kanidm_client/src/lib.rs

```diff
@@ -292,7 +295,7 @@ impl KanidmClientBuilder {
         let address = match &self.address {
             Some(a) => a.clone(),
             None => {
-                eprintln!("uri (-H) missing, can not proceed");
+                eprintln!("Configuration option 'uri' missing, cannot continue client startup.");
```

Comment:
Should this be error!?

Reply:
Probably if we had a single way of logging ;)

Reply:
Ahhh yeah, that. Okay, well in that case all good.